jormungandr/src/explorer/indexing.rs

```diff
@@ -104,12 +106,21 @@ pub struct ExplorerVotePlan {
     pub proposals: Vec<ExplorerVoteProposal>,
 }
 
+#[derive(Clone)]
+pub enum ExplorerVote {
+    Public(Choice),
+    Private {
+        proof: ProofOfCorrectVote,
+        encrypted_vote: EncryptedVote,
+    },
+}
+
 #[derive(Clone)]
 pub struct ExplorerVoteProposal {
     pub proposal_id: ExternalProposalId,
     pub options: Options,
     pub tally: Option<ExplorerVoteTally>,
-    pub votes: Hamt<ExplorerAddress, Choice>,
```

Comment:
This is yet another non-scalable votes member, should be converted to a paginated connection.

Reply:
Makes sense. Will do.

Reply:
Filed as #2747.